### ISSUE TITLE
Support non-localhost address in the frontend

### ISSUE THREAD

**Issue by gz-c** [skycoin/skycoin#1939]
(Split from #1258)

The frontend hardcodes localhost as the service address; support an arbitrary service address so that the UI can be viewed remotely.

Related: #1902

**Comment by Senyoret1** [comment]
I did some preliminary testing, but I could not get very far. At first I could only access the UI with http://127.0.0.1:6420, when trying to access with http://localhost:6420 the server returned errors, but it was solved with `-host-whitelist=localhost:6420`. However, when trying to use the IP of my own PC there was no response. For example, if the IP of my PC is 192.100.0.1, calls to http://192.100.0.1:6420 do not return any type of response. This, of course, also implies that there is no response whe trying from another PC in the same network.

Does the server used by the node allow connections other than from Localhost?

**Comment by gz-c** [comment]
With the `-host-whitelist` option, the server should allow connections from other than localhost. Even without the `-host-whitelist` option, you should get a response from non-localhost addresses, but it will be an error response.

**Comment by Senyoret1** [comment]
Sorry for not trying again before. I did put my ip and port in `-host-whitelist` and the result was the same, no response at all. This is the Postman output using localhost:
![response2](https://user-images.githubusercontent.com/34079003/47366452-b76e2200-d6ab-11e8-9d8d-4db378f0404b.png)
And this is the postman output using the local IP:
![response1](https://user-images.githubusercontent.com/34079003/47366476-c1902080-d6ab-11e8-88ee-410a2aa12e0d.png)
Postman does not show headers or anything else. To verify if there was a problem with the IP, I tried with a web server running on port 80 and it responded correctly to the requests. I do not know if it could be a local problem or something with Windows.
